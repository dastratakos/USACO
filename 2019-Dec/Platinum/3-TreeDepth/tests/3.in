7 15 920024579
