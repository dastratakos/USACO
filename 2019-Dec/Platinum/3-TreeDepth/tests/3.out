913 928 891 825 735 620 467
